src/components/core/ApplicationHeader.tsx

```diff
@@ -159,6 +159,13 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
 								/>
 							</div>
 						</div>
+
+						{/* Report Problem */}
+						<div className="relative cursor-pointer">
+							<Link href="https://forms.office.com/r/z48ExG8dPs">
+								<UnstyledButton className="w-40 h-full p-2">Report Problem</UnstyledButton>
```

Comment:
Why UnstyledButton? Use the outline variant of the Button component

Reply:
For references @KongPobW 
https://mantine.dev/core/button/